cypress/support/commands.js

```diff
@@ -1,16 +1,14 @@
-import loginPage from '../pages/loginPage';
-import checkoutInformationPage from '../pages/checkoutInformationPage';
-const LoginPage = new loginPage();
-const CheckoutInformationPage = new checkoutInformationPage();
+import LoginPage from '../pages/LoginPage';
+import CheckOutInformationPage from '../pages/CheckOutInformationPage';
 
-Cypress.Commands.add("authentication", function (userName) {
-    cy.fixture("users").then((userData)=>{
+Cypress.Commands.add('authentication', (userName) => {
```

Comment:
Did you test and run the project with this modification?
function() preserves the context of his body meanwhile arrow function doesn't do that